index.html

```diff
@@ -4,10 +4,312 @@
+                </svg>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="travel-ad-up travel" id="what_we_do">
+        <div class="container">
+          <div class="row">
+            <div class="col-12 col-lg-7 col-xl-6">
+              <div class="travel-ad-up__card card">
+                <div class="card__title--small">
+                  <p class="title--small">spain</p>
+                </div>
+                <div class="card__title">
+                  <h5 class="title-5">Grand Spain Madrid</h5>
+                </div>
+                <div class="card__text text">
+                  But I must explain to you how all this mistaken idea of
+                  account to denouncing pleasure and praising pain was account
+                  born and I account will give you a complete account of the
+                  system
+                </div>
+                <div class="card__button button">book now</div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="travel-ad-down travel" id="travel">
+        <div class="container">
+          <div class="row">
+            <div class="col-12 col-lg-7 offset-lg-5 col-xl-6 offset-xl-6">
+              <div class="travel-ad-down__card card">
+                <div class="card__title--small">
+                  <p class="title--small">spain</p>
+                </div>
+                <div class="card__title">
+                  <h5 class="title-5">Grand Spain Madrid</h5>
+                </div>
+                <div class="card__text text">
+                  But I must explain to you how all this mistaken idea of
+                  account to denouncing pleasure and praising pain was account
+                  born and I account will give you a complete account of the
+                  system
+                </div>
+                <div class="card__button button">book now</div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="what">
+        <div class="what__title">
+          <h2 class="title-2">What Our Tourists Say</h2>
+        </div>
+        <div class="container">
+          <div class="row">
+            <div class="col-12 col-lg-4 m-0 p-0">
+              <div class="what__img"></div>
+              <img
+                class="what__img--mobile"
+                src="img/ReviewPhoto.png"
+                alt="tourist looking at a mountain scenery"
+              />
+            </div>
+            <div class="col-12 col-lg-8 m-0 p-0">
+              <div class="what__card card">
+                <div class="what__card-title">
+                  <h5 class="title-5">Sam Sumon</h5>
+                </div>
+                <div class="what__text">
+                  <blockquote class="text">
+                    <sup> <i class="icon-quote-left what__card-quote"></i></sup>
```

Comment:
Краще зробити блок **quote** і лапки додати через  `::before`
```suggestion
                  <blockquote class="quote text">                    
```